### ISSUE TITLE
Action required from 1.4 feature owners - update release notes spreadsheet

### ISSUE THREAD

**Issue by pwittrock** [kubernetes/kubernetes#33226]
>
Note: Any messages to this github issue will go to many many people, so there is no need to update it unless you need clarification on the instructions.

If you are mentioned in the issue, you must **make sure the 1-line feature description for your 1.4 issues has been updated**.

We are aggregating release notes for the kubernetes 1.4 release and need a 1-line description + docs link for each feature.

Please update the following columns in [this spreadsheet](https://docs.google.com/spreadsheets/d/1MeTQbtSiTCoQ74zjEGRI1rXRSP0Tzmo9-kMJSK_HGoA/edit?ts=57e2d4ba#gid=0) with:
- "One Line Release Note Description": A 1-line description of your feature 
- "Link To Docs": A link to the doc PR or where they will be published on the main site
- "Comments": Any special notes about the feature that @spiffxp should know about when he aggregates the data in this spreadsheet.

If your feature should not appear in the release notes summary section, please remove the row for it from the spreadsheet.

*Thank you for your sustained efforts and apologies for the spam if you have already gotten this message or updated the spreadsheet.*

@davidopp 
- Prioritized scheduling of critical cluster addon pods
- Workload spreading across failure domains (fix pod anti-affinity performance problem))

@piosz
- Prioritized scheduling of critical cluster addon pods

@philips 
- Container Image Policy
-Kubelet TLS Bootstrap

@dchen1107
- Automate Docker Validation Process 

@wojtek-t
- Workload spreading across failure domains (fix pod anti-affinity performance problem))

@derekwaynecarr
- kubelet evicts resources when observing disk pressure

@deads2k
- Access Review APIs

@childsb
- Configurable Dynamic Provisioning aka StorageClass

@sttts
- Add sysctl support

@soltysh 
- ScheduledJobs

@jbeda 
- Dramatically Simplify Kubernetes Cluster Creation

@pweil- 
- Pod Security Policy

**Comment by idvoretskyi** [comment]
cc @kubernetes/features-maintainers to track.

**Comment by spiffxp** [comment]
@kubernetes/docs a lot of people are providing links to PR's to the docs site, which would be tough for an end-user to read through

what sorts of links should we use to get to the staged results of their PR's being merged?

eg: where I could see a live copy of https://github.com/kubernetes/kubernetes.github.io/pull/1206

**Comment by spiffxp** [comment]
There are some feature issues in the v1.4 milestone that aren't on the spreadsheet.  Pinging the authors/assignees to confirm you do _not_ want these mentioned in the release notes
- (alpha) [Node Conformance Test](https://github.com/kubernetes/features/issues/84) (@dchen1107, @Random-Liu)
- (alpha) [Node Performance Benchmark](https://github.com/kubernetes/features/issues/83) (@dchen1107, @dchen1107, @coufon)
- (stable) [New Volume Plugin: Quobyte](https://github.com/kubernetes/features/issues/80) (@saad-ali, @rootfs)
- (stable) [New Volume Plugin: Azure Data Disk](https://github.com/kubernetes/features/issues/79) (@saad-ali, @rootfs)

**Comment by soltysh** [comment]
@pwittrock I've added audit https://github.com/kubernetes/features/issues/22 to that spreadsheet, since it's marked as alpha for 1.4, feel free to remove if not needed.

**Comment by idvoretskyi** [comment]
@soltysh I've been preparing the spreadsheet and haven't included https://github.com/kubernetes/features/issues/22 due to the "Milestone 1.5".

Anyway, if this feature is going to be included into the release, thank you for adding it.

**Comment by soltysh** [comment]
@idvoretskyi it's not, because the full functionality is targeted for 1.5, but some basic functionality is already available in 1.4 (thus the alpha-1.4 label on it). There are some docs in place already so I've decided to include it as well.

**Comment by idvoretskyi** [comment]
@soltysh thank you!

**Comment by piosz** [comment]
cc @fgrzadkowski since @wojtek-t is OOO

**Comment by spiffxp** [comment]
@devin-donnelly I notice there is a https://kubernetes-v1-4.github.io/ but it doesn't seem to have all of the latest files, or I'm not understanding how to convert the PR's merged into links to that site.  

Was hoping to provide links to something like that, such that once docs are deployed to the main site, it's a regex/replace to update release note doc links

FWIW, the specific example I linked failed its netlify deploy.  Where I could I see the results of https://github.com/kubernetes/kubernetes.github.io/pull/1206 ?

**Comment by devin-donnelly** [comment]
One of the previous PRs overwrote some of the required configuration files.
Any PR that's failed its Netlify deploy needs to rebase.

We're not using "kubernetes-v1-4.github.io" as staging anymore. It was a
hack that created a dummy organization and copied the entire repo out using
a Travis script that was continuously broken for 90% of docs PRs.

You can link directly to the deployed Netlify stuff, and just replace the
subdomain once you've deployed, if you want to do that.

**Comment by spiffxp** [comment]
@devin-donnelly ok thanks for clarifying, what do we do for PR's that have already been merged, such as the example I linked?  will some later netlify deploy just pick that up?

**Comment by devin-donnelly** [comment]
I think I can just run a commanded deployment that should pick up
everything in the branch (and thus everything that's been merged to date).
I'll send you the link to the subdomain where Netlify stages it and it
should contain all the changes.

Stand by.

**Comment by spiffxp** [comment]
@madhusudancs please provide docs links for the various Federation features

**Comment by devin-donnelly** [comment]
Okay, got it.

All the latest release-1.4 docs are available here:
http://kubernetes-io-vnext-staging.netlify.com/docs/

I've spot-checked for the presence off merged PRs. This one was an easy
check: https://github.com/kubernetes/kubernetes.github.io/pull/1147/
The AppArmor documentation is present at the staging URL.

Let me know if that helps.

**Comment by jbeda** [comment]
@lukemarsden -- I added you as a contact for the "simplify k8s install" item.  Can you put a link to docs there once they land?

**Comment by madhusudancs** [comment]
> @madhusudancs please provide docs links for the various Federation features

@spiffxp: @pwittrock and I discussed about this yesterday and decided to leave those fields until we know where the docs are going to end up.

I did some digging today based on @devin-donnelly's [netlify](http://kubernetes-io-vnext-staging.netlify.com/docs/) link above. I have now populated the links to the docs for which I know where they are going to end up. I will update the fields again as soon as I figure out where the remaining ones are going to end up.

**Comment by devin-donnelly** [comment]
That link I provided above should automatically updated as release-1.4 PRs
are merged. You can think of it as the dedicated staging site.

The links attached to individual PRs should provide a preview for any
as-yet-unmerged PRs.

**Comment by madhusudancs** [comment]
@devin-donnelly 

> That link I provided above should automatically updated as release-1.4 PRs are merged. You can think of it as the dedicated staging site.

Sure. I am not using the host part of that URL at all. I am doing a s/kubernetes-io-vnext-staging.netlify.com/kubernetes.io/ on the host part. But I still don't know the paths of some of these docs at all.

**Comment by calebamiles** [comment]
@pwittrock, I'll be taking the issues assigned to @philips, oneliner release descriptions are done; docs are merged; we're happy to get you anything else you need for the release.

**Comment by spiffxp** [comment]
@dchen1107 re: https://github.com/kubernetes/features/issues/57 it would be preferable to see some docs around what the automated docker validation process is, but I already want so many ponies

can you please clarify which versions of docker have been qualified? are other container runtimes qualified?
